pallets/regulated-assets/src/lib.rs

```diff
@@ -154,37 +171,56 @@ pub mod pallet {
         /// ## Parameters
         ///
         /// - `origin`: The origin of the transaction.
-        /// - `symbol`: The symbol of the SBT which should represent string with only uppercase latin chars with max length of 7.
-        /// - `name`: The name of the SBT should represent string with only uppercase or lowercase latin chars or numbers or spaces, with max length of 33.
-        /// - `allowed_assets`: TThe list of assets allowed to be operated with by holding the SBT.
+        /// - `symbol`: The symbol of the SBT which should represent a string with only uppercase Latin characters with a maximum length of 7.
+        /// - `name`: The name of the SBT which should represent a string with only uppercase or lowercase Latin characters, numbers, or spaces, with a maximum length of 33.
         /// - `description`: The description of the SBT. (Optional)
+        /// - `image`: The URL or identifier for the image associated with the SBT. (Optional)
+        /// - `external_url`: The URL pointing to an external resource related to the SBT. (Optional)
+        /// - `expires_at`: The timestamp indicating when the SBT expires. (Optional)
+        /// - `allowed_assets`: The list of assets allowed to be operated with by holding the SBT.
         #[pallet::call_index(1)]
         #[pallet::weight(<T as Config>::WeightInfo::issue_sbt())]
         pub fn issue_sbt(
             origin: OriginFor<T>,
             symbol: AssetSymbol,
             name: AssetName,
-            allowed_assets: BoundedVec<AssetIdOf<T>, T::MaxAllowedTokensPerSBT>,
             description: Option<Description>,
+            image: Option<ContentSource>,
+            external_url: Option<ContentSource>,
+            expires_at: Option<T::Moment>,
+            allowed_assets: BoundedVec<AssetIdOf<T>, T::MaxAllowedAssetsPerSBT>,
         ) -> DispatchResult {
             let who = ensure_signed(origin)?;
 
-            // Check permission `who` can issue SBT
-            Permissions::<T>::check_permission(who.clone(), ISSUE_SBT)?;
+            let now_timestamp = Timestamp::<T>::now();
+            if let Some(expires_at) = expires_at {
+                ensure!(
+                    expires_at > now_timestamp,
+                    <Error<T>>::SBTExpirationDateCannotBeInThePast
+                );
+            }
+
+            Self::check_allowed_assets_for_sbt_issuance(&allowed_assets, &who)?;
 
             let sbt_asset_id = T::AssetManager::register_from(
                 &who,
                 symbol,
                 name.clone(),
-                DEFAULT_BALANCE_PRECISION,
+                0,
                 0,
                 true,
-                None,
+                image.clone(),
                 description.clone(),
             )?;
+
             let metadata = SoulboundTokenMetadata {
+                owner: who.clone(),
                 name,
                 description,
+                image: image.clone(),
```

Comment:
Do we really need to duplicate owner, image and description here? We already have it in AssetInfo

Reply:
No, we don't need it. I thought of the case to ease retrieving data, but I was wrong.
I will remove name, description, image, and description fields to not duplicate info.

ok? @vovac12